src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let rentCar = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    rentCar -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
4. [checklist] Prefer `if` with `return` over `if else` to simplify later conditions.